Make package more dynamic

xV1CT0Rx opened the pull request:
- I have never made/updated a Choco package, so I couldn't test everything completely, so before this gets committed to anything please VERIFY that it's working as intended
- If I made some general

<!-- Provide a general summary of your changes in the Title above, prefixed with (packageName) -->

## Description
<!-- Describe your changes in detail -->
- Updated chocolateyInstall.ps1 to download all needed files online (not from choco package)
- Removed Certificate (gets downloaded in chocolateyInstall.ps1)
- Removed verification (since no setup files are directly in the Choco package anymore, we don't need to verify anything)
- Updated the .nuspec file with the current package version number

## Motivation and Context
<!-- Why is this change required? What problem does it solve? -->
Makes the VirtualBox package more dynamic by downloading all required files directly from the vendor website instead of putting them in the Chocolatey package
<!-- If it fixes an open issue, please link to the issue here. -->
<!-- Use fixes/fixed when referencing the issue -->

## How Has this Been Tested?
<!-- Please describe in detail how you tested your changes. -->
I was only able to test the bare minimum, as I have never made or updated a Choco package, so the NEEDS to be tested before committing it to anything
<!-- Include details of your testing environment, and the tests you ran to -->
<!-- see how your change affects other areas of the script, etc. -->

## Screenshot (if appropriate, usually isn't needed):

## Types of changes
<!-- What types of changes does your code introduce? Put an `x` in all the boxes that apply: -->
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
<!-- Go over all the following points, and put an `x` in all the boxes that apply. -->
<!-- If you're unsure about any of these, don't hesitate to ask. We're here to help! -->
- [ ] My code follows the code style of this repository (if your change does not follow the style it will likely be rejected - this includes changes in formatting / layout).
- [ ] My change requires a change to the documentation (this usually means the notes in the description of a package) and I have updated the documentation accordingly.
- [ ] I have updated the package description and it is less than 4000 characters.
- [ ] The added / modified package passed install / uninstall in the chocolatey test environment.
I don't have a test environment yet
- [x] The changes only affect a single package (not including meta package).

pauby commented on automatic/virtualbox-guest-additions-guest.install/tools/chocolateyInstall.ps1:
```diff
@@ -1,45 +1,46 @@
+	$certFileName = 'Trusted-OracleCorporationVirtualBox-05308b76ac2e15b29720fb4395f65f38.cer'
+	$certUrl      = "https://www.virtualbox.org/svn/vbox/trunk/src/VBox/HostDrivers/Support/Certificates/$certFileName"
+	$certFile     = Join-Path $unzipLocation $certFileName
+	$WebClient    = [System.Net.WebClient]::new()
```
You can't do this as part of a Chocolatey package on the Chocolatey Community Repository.

xV1CT0Rx replied:
Just for me for clarification, do you mean because this part doesn't have something to do directly with Chocolatey?

pauby commented on automatic/virtualbox-guest-additions-guest.install/tools/chocolateyInstall.ps1:
```diff
@@ -1,45 +1,46 @@
+	$packageArgs = @{
+		packageName    = $ENV:ChocolateyPackageName
+		fileType       = 'EXE'
+		file           = Join-Path $unzipLocation $binX86Filename
+		file64         = Join-Path $unzipLocation $binX64Filename
```
You haven't added the checksums for the binaries. These are also needed for the Chocolatey Community Repository.

pauby commented on automatic/virtualbox-guest-additions-guest.install/virtualbox-guest-additions-guest.install.nuspec:
```diff
@@ -26,7 +26,7 @@ This is a nuspec. It mostly adheres to https://docs.nuget.org/create/Nuspec-Refe
         <!-- Is the version a prerelease of a version? https://docs.nuget.org/create/versioning#creating-prerelease-packages -->
         <!-- Note that unstable versions like 0.0.1 can be considered a released version, but it's possible that one can release a 0.0.1-beta before you release a 0.0.1 version. If the version number is final, that is considered a released version and not a prerelease. -->
-        <version>6.1.18</version>
+        <version>6.1.36</version>
```
This version already exists in the Chocolatey Community Repository.

xV1CT0Rx replied:
Really, the latest I can see is 6.1.34

pauby commented:
Thanks for the PR. I'm trying to understand - what issue does this solve? Is the certificate that is part of the package, invalid?

xV1CT0Rx commented:
Thanks for your quick replies.
The certificate in the package isn't invalid, I just removed it from the package and added it as download directly from the manufacturers website, it's exactly the same certificate

I was generally trying to make the package more straightforward and dynamic by including everything needed as download links instead of files in the package, just like I've seen with most other Chocolatey packages, where it also downloads the software directly from the manufacturers website.
